meta-flutter/classes/embedded-linux.bbclass

```diff
@@ -0,0 +1,26 @@
+SUMMARY = "Common parts of the recipe for flutter-embedded-linux"
+AUTHOR = "Hidenori Matsubayashi"
+HOMEPAGE = "https://github.com/sony/flutter-embedded-linux"
+BUGTRACKER = "https://github.com/sony/flutter-embedded-linux/issues"
+LICENSE = "BSD 3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=79ca841e7b9e09b0401186f2aa334adf"
```

Comment:
LICENSE file isn't exist.

Reply:
It is looking at `https://github.com/sony/flutter-embedded-linux/blob/master/LICENSE`.